components/CheckoutButton/CheckoutButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const CheckoutButton = () => (
+  <React.Fragment>
```

Comment:
Could you take `({ children, ...otherProps })` and instead of hard coded text put children and spread `otherProps` at root component (`a`)?  This way we can bind event listeners, classNames etc.

Reply:
Alright, will do this tonight :)

Reply:
Any update?